lsp/markdown/completion/cron.md

```diff
@@ -0,0 +1,18 @@
+Snippet for declaring a cron job.
+
+```go
+//ftl:cron 0 * * * *
+func Hourly(ctx context.Context) {}
+
+//ftl:cron 6h
+func EverySixHours(ctx context.Context) {}
+```
+
+See https://tbd54566975.github.io/ftl/docs/reference/cron/
+---
+
+//ftl:cron ${1:Schedule}
```

Comment:
I think it might be better to split this snippet into two? One where we explicitly list each component of the full form, eg. `${1:minute} ${2:hour} ...` and one for the short form.